common/pkg/capi/clustertopology/variables/enums.go

```diff
@@ -4,22 +4,15 @@
 package variables
 
 import (
-	"encoding/json"
-	"fmt"
-
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 )
 
-func ValuesToEnumJSON[T any](vals ...T) ([]apiextensionsv1.JSON, error) {
+func MustMarshalValuesToEnumJSON[T any](vals ...T) []apiextensionsv1.JSON {
```

Comment:
Good change

Reply:
Thank you @dkoshkin, that means a lot :wink:

Reply:
😆